sdks/solana/stork-sdk/README.md

```diff
@@ -0,0 +1,66 @@
+# Stork Solana SDK
+
+This is a Rust SDK to build Solana programs that consume Stork price feeds. This crate is maintained by [Stork Labs](https://stork.network).
+
+It is available on [crates.io](https://crates.io/crates/stork-sdk).
+
+## Pull Model
+
+The Stork Solana SDK allows users to consume Stork price updates on a pull basis. This puts the responsibility of submitting the price updates on-chain to the user whenever they want to interact with an app that consumes Stork price feeds. Stork Labs maintains a [Chain Pusher](https://github.com/stork-oracle/stork-external/apps/docs/chain_pusher) in order to do this.
+
+## Stork Price Feed Accounts
+
+On Solana, Stork price feeds exist as on-chain accounts. These accounts are instances of the [`TemporalNumericValueFeed` account](./src/temporal_numeric_value.rs), and are created and owned by the Stork Oracle contract. These account have an ID which associates them with a specific asset, and a `latest_value` field which stores the latest price update.
+
+## Example
+
+The following is an example of how to use the Stork Solana SDK to consume Stork price updates, available [here](https://github.com/Stork-Oracle/stork-external/tree/main/examples/solana).
+
+```rust
```

Comment:
I'm not a huge fan of including so much context in code here as there will likely be drift between these docs and the actual code itself. If you want to do a code snippet, I would limit to much less information and still link the example

Reply:
I slimmed down the snippet a little bit. This readme is all the crates.io landing page for the crate, and having some sort of snippet is pretty standard.